### PR TITLE
Add python3 support

### PR DESCRIPTION
**Pull request by jacobgardner**
Bunch of stuff that causes locustio to pass all tests in python2 and python3. 

Let me know if you find anything that isn't working.

### CODE REVIEW

**Comment by pmdarrow** [comment]
Looks like there are a lot of whitespace changes. You should probably move those into a separate PR, it makes it really hard to review.

File: setup.py [hunk 1]
```diff
@@ -47,7 +47,7 @@ def run(self):
     packages=find_packages(exclude=['ez_setup', 'examples', 'tests']),
     include_package_data=True,
     zip_safe=False,
-    install_requires=["gevent>=1.0.1", "flask>=0.10.1", "requests>=2.4.1", "msgpack-python>=0.4.2"],
+    install_requires=["gevent>=1.0.1", "flask>=0.10.1", "requests>=2.4.1", "msgpack-python>=0.4.2", "future>=0.15.0", "six>=1.9.0"],
```
Comment (pmdarrow) [inline setup.py]: I don't think gevent 1.0.1 is compatible with Python 3. You need >=1.1, see https://github.com/gevent/gevent/blob/master/changelog.rst#11a1-jun-29-2015.

**Comment by jacobgardner** [comment]
@pmdarrow, I will try to make a commit reverting all the whitespace changes tomorrow.  

In the meantime, if you want, you can add ?w=1 to the end of the URL, github will show the diff with only non-whitespace changes.

Thanks!

**Comment by jgentil** [comment]
Has this been merged into master yet?  I have a dependency on this happening in order to use locust with Python 3.

**Comment by pmdarrow** [comment]
This has not been merged. Can any core maintainers comment on this? It looks like this PR has been sufficiently cleaned up.

**Comment by jmelfi** [comment]
Any ETA here on when this can be merged?  Dependency also here that needs 3 support if possible.

**Comment by pmdarrow** [comment]
It appears this project is no longer being actively maintained (see [commits for 2015](https://github.com/locustio/locust/graphs/contributors?from=2015-01-01&to=2015-10-24&type=c)). We might be better served by maintaining our own Python 3 fork.

**Comment by jmelfi** [comment]
@pmdarrow thanks for that.  I was hoping for a last fix so I didn't have to do it myself.  Might have to keep using Gatling since that's currently running already in our infrastructure.

Best!

**Comment by pmdarrow** [comment]
Cleaned things up a little bit more in https://github.com/locustio/locust/pull/363 and pinged the repo owners — hopefully we can get this merged!!

**Comment by justiniso** [comment]
Closing this in favor of https://github.com/locustio/locust/pull/363, which is being actively developed.

**Pull request closed without merging**